docs/tutorials/qml/config_qnn.md

```diff
@@ -0,0 +1,111 @@
+
+## Defining the Feature Map
+
+Let us say we want to build a 4-qubit QNN that takes two inputs, namely, the $x$ and the $y$ coordinates of a point in the plane. We can use the `FeatureMapConfig` class to specify the feature map.
+
+```python exec="on" source="material-block" session="config" html="1"
+from qadence.ml_tools.config import FeatureMapConfig
+from qadence.types import BasisSet, ReuploadScaling
+from qadence.blocks import chain
+from qadence.ml_tools.constructors import create_fm_blocks
+
+fm_config = FeatureMapConfig(
+    num_features=2,
+    inputs = ["x", "y"],
+    basis_set=BasisSet.CHEBYSHEV,
+    reupload_scaling=ReuploadScaling.TOWER,
+    feature_range={
+        "x": (-1.0, 1.0),
+        "y": (0.0, 1.0),
+    },
+)
+
+fm_blocks = create_fm_blocks(register=4, config=fm_config)
+feature_map = chain(*fm_blocks)
+from qadence.draw import html_string # markdown-exec: hide
+print(html_string(feature_map)) # markdown-exec: hide
+```
+
+We have specified that the feature map should take two features, and have named the [`FeatureParameter`][qadence.parameters.FeatureParameter] "x" and "y" respectively. Both these parameters are encoded using the Chebyshev basis set, and the reupload scaling is set to `ReuploadScaling.TOWER`. One can optionally the basis and the reupload scaling for each parameter separately.
+
+The `feature_range` parameter is a dictionary that specifies the range of values that each feature comes from. This is useful for scaling the input data to the range that the encoding function can handle. In default case, this range is mapped to the target range of the Chebyshev basis set which is $[-1, 1]$. One can also specify the target range for each feature separately.
+
+For full details on the `FeatureMapConfig` class, see the [API documentation][qadence.ml_tools.config.FeatureMapConfig].
+
+## Defining the Ansatz
+
+The next part of the QNN is the ansatz. We use `AnsatzConfig` class to specify the type of ansatz.
+
+Let us say, we want to follow this feature map with 2 layers of hardware efficient ansatz.
+
+```python exec="on" source="material-block" session="config" html="1"
+from qadence.ml_tools.config import AnsatzConfig
+from qadence.ml_tools.constructors import create_ansatz
+
+ansatz_config = AnsatzConfig(
+    depth=2,
+    ansatz_type="hea",
+    ansatz_strategy="digital",
+)
+
+ansatz = create_ansatz(register=4, config=ansatz_config)
+
+print(html_string(ansatz)) # markdown-exec: hide
+```
+
+We have specified that the ansatz should have a depth of 2, and the ansatz type is "hea" (Hardware Efficient Ansatz). The ansatz strategy is set to "digital", which means digital gates are being used. One could alternatively use "analog" or "rydberg" as the ansatz strategy.
```

Comment:
This was to get around the fact that "Rydberg" did not exist as a strategy. But as @jpmoutinho suggested above, I will add that and make the changes accordingly.

It is just that "Rydberg" isn't strictly a "strategy" like sDAQC, bDAQC and so on are currently in the StrEnum. But since there is a use for it, I could just go ahead and add it.

Reply:
Yes I thought a bit whether to suggest that or not because like you said it's a bit against what the initial purpose of the strategy enum was. But yeah for now it's useful so let's just add it.